bootstrap_cfn/elb.py

```diff
@@ -0,0 +1,75 @@
+        else:
+            print "[ERROR] No AWS credentials"
+            sys.exit(1)
+        self.iam = iam.IAM(config)
+        self.cfn = cloudformation.Cloudformation(config)
+
+    def set_ssl_certificates(self, ssl_config, stack_name):
+        """
+        Look for SSL listeners on all the load balancers connected to
+        this stack, then set update the certificate to that of the config
+
+        Args:
+            ssl_config (dictionary): Certification names to corresponding data
+            stack_name (string): Name of the stack
+        """
+        for cert_name in ssl_config.keys():
+            # Get the cert id and also its arn
+            cert_id = "{0}-{1}".format(cert_name, stack_name)
+            cert_arn = self.iam.get_arn_for_cert(cert_id)
+
+            # Get all stack load balancers
+            load_balancer_resources = self.cfn.get_stack_load_balancers(stack_name)
+            found_load_balancer_names = []
+            for load_balancer_resource in load_balancer_resources:
+                found_load_balancer_names.append(load_balancer_resource.physical_resource_id)
+
+            # Use load balancer names to filter gettingload balancer details
+            if len(found_load_balancer_names) > 0:
+                load_balancers = self.conn_elb.get_all_load_balancers(load_balancer_names=found_load_balancer_names)
+
+            # Look for https listeners on load balancers and update the cert
+            # using the arn
+            if len(load_balancers) > 0:
+                for load_balancer in load_balancers:
+                    for listener in load_balancer.listeners:
+                        # Get protocol, if https, update cert
+                        # in_port = listener[0]
+                        out_port = listener[1]
+                        protocol = listener[2]
+                        # If the protocol is HTTPS then set the cert on the listener
+                        if protocol == "HTTPS":
+                            logging.info("ELB::set_ssl_certificates: "
+                                         "Found HTTPS protocol on '%s', "
+                                         "updating SSL certificate with '%s'" 
+                                         % (load_balancer.name, cert_arn))
+                            self.conn_elb.set_lb_listener_SSL_certificate(load_balancer.name,
+                                                                          out_port,
+                                                                          cert_arn
+                                                                          )
+            else:
+                # Just info, there being no load balancers to update is not
+                # necessarily a problem
+                logging.info("ELB::set_ssl_certificates: "
+                             "No load balancers found in stack, "
+                             "no certificate updates needed")
```

Comment:
This probably be a warning or an error - the user has requested something to be updated that couldn't be. This also means we should likely return a non-0 exit code.